packages/components/src/avatar/docs/Avatar.stories.mdx

```diff
@@ -68,13 +68,11 @@ An avatar component can vary in size.
 <Preview scope={{ ChrisHadfield }}>
```

Comment:
It looks kind of cool, but I'm guessing that they're not supposed to be this close? The old one has space in between.
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/31b7ded5-d449-483b-81d2-dd70ad2fe1c5)

Reply:
I based the rebrand on Hopper's Figma and had to adapt it to what exists in Orbit.

https://www.figma.com/file/zSpuR1wOAT99H6YfD29IRl/%F0%9F%90%B8-Hopper-Design-System?type=design&node-id=31%3A125&mode=dev

Reply:
Oh alright then it's good escept.. the +2 has a grey background here, which it doesn't have in figma from what I can tell